pytorch_lightning/utilities/apply_func.py

```diff
@@ -155,6 +155,14 @@ def batch_to(data):
             return device_data
 
         kwargs = dict(non_blocking=True) if isinstance(data, torch.Tensor) else {}
+
+        if _PYSYFT_AVAILABLE:
+            from syft.core.pointer.pointer import Pointer
+            from syft import client_cache
+            if isinstance(data, Pointer):
+                data = data.get(request_block=True, delete_obj=False)
+                data = data.send(client_cache["duet"])
```

Comment:
why do we need to send here?

Reply:
No idea. That's from OpenMined. @tudorcebere

Reply:
It is used to cash data from my understanding.

Reply:
It works as a cast, we will come with this mechanism as soon as possible, but we want to get the real pointer type here.